core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -602,7 +602,11 @@ maybe_erase_changed(Db, 'database', _Id, Tab) ->
     erase_changed(Objects, [], Tab);
 
 maybe_erase_changed(Db, Type, Id, Tab) ->
-    MatchSpec = [{#cache_obj{origin = {'db', Db}, _ = '_'}
+    MatchSpec = [{#cache_obj{origin = {'database', Db}, _ = '_'}
```

Comment:
and there's actually a missed cache happening on database delete. i'll add that do the branch.